src/main/scala/firrtl/backends/experimental/smt/random/DefRandom.scala

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package firrtl.backends.experimental.smt.random
+
+import firrtl.ir._
+
+/** Named source of random values. If there is no clock expression, than it will be clocked by the global clock. */
+case class DefRandom(
```

Comment:
Alternative names that I would be fine with: `DefAny`, `DefArbitrary`, `DefAnySeq`

I don't want to use the term `Poison` since that is LLVM terminology for something similar but not quite the same as what we are talking about here. So I believe there is too much room for confusion.